src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1157,7 +1157,8 @@ static bool processContextAttributeVector (ContextElement*
                                            double&                                    coordLat,
                                            double&                                    coordLong,
                                            std::string                                tenant,
-                                           const std::vector<std::string>&            servicePathV)
+                                           const std::vector<std::string>&            servicePathV,
+                                           std::string*                               why)
```

Comment:
References should work also to send information back, but it is ok to use a pointer if you prefer so. Thus, NTC after dcbbe54.